vite.config.ts

```diff
@@ -3,12 +3,14 @@ import react from '@vitejs/plugin-react';
 import { resolve } from 'node:path';
 import { defineConfig } from 'vite';
 import dts from 'vite-plugin-dts';
+import svgLoader from 'vite-svg-loader';
 import tsConfigPaths from 'vite-tsconfig-paths';
 import { name } from './package.json';
 
 export default defineConfig(() => ({
   plugins: [
     eslintPlugin(),
+    svgLoader(),
```

Comment:
Adding this `defaultImport` format allows SVG import without specifying the import type inline. 
 
```suggestion
    svgLoader({ defaultImport: 'url' }),
```